lib/index.js

```diff
@@ -86,7 +86,12 @@ YeomanTest.prototype.testDirectory = function (dir, cb) {
 
   try {
     if (fs.existsSync(dir)) {
-      fs.rmdirSync(dir, {recursive: true});
+      const args = [dir, {recursive: true}];
+      if (fs.rmSync) {
+        fs.rmSync(...args);
+      } else {
+        fs.rmdirSync(...args);
+      }
```

Comment:
```suggestion
      // Fallback 'rmSync' to 'rmdirSync' for node < 14.14.0
      const { rmdirSync, rmSync = rmdirSync } = fs;
      rmSync(dir, {recursive: true});
```

Would be better to move the 'rmSync' definition to the require statement or after the imports for visibility and reuse.

Reply:
Oh, this is nice! Applied!